docs/catalog/web/minimize-main-thread-work.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: web
+tags: 
+ - web
+ - role:web-developer
```

Comment:
does this need a compute tag as well since it is about how work is being processed?